### PR TITLE
git add readme

### PR DESCRIPTION
**Pull request by evgenybritan**
Fixed issues:
1. g=HB0?
2. please move it before switch ( arr= arr_type[pflag]; )
3. pflag=Packet_type(HB0); can be done once before switch
4. useless. Seems that path is redundant variable (path=get_path;)
5. Please use readable defines ( It is not clear what is 0x80?
Avoid to use in code just numbers. Please define then with macros or variables.)
6. Why name get_path but not just path?\
7. Better to declare this enum in header
8. why? ( using namespace std;)
9. Remove unused includes
10. Private variables better to starts with "_"
11. it is better to describe first public code
12. Class name should starts with capital letter and equal to file name. It is better to name this class as Parser
13. Do not use hardcoded filenames (that issue is old)
14. Please wrap you code with cpp classed ( is old too)

### CODE REVIEW

**Comment by Java8Test** [comment]
Do not be lazy to write more informative comments on commit )

**Comment by evgenybritan** [comment]
Ok, I will be more informative in such things.

File: converter/parser.h [hunk 1]
```diff
@@ -2,20 +2,32 @@
...
 
-#include <fstream>
 #include <iostream>
-#include <stdlib.h>
 #include <QString>
 
-using namespace std;
-
-class parser_packet
+enum Packet_type{
```
Comment (Java8Test) [inline converter/parser.h]: Please, move it inside the class

File: converter/parser.h [hunk 2]
```diff
@@ -2,20 +2,32 @@
 #define PARSER_H
 
 
-#include <fstream>
 #include <iostream>
```
Comment (Java8Test) [inline converter/parser.h]: Why do you need iostream here?

File: converter/parser.h (hunk 1 above)
Comment (evgenybritan) [reply converter/parser.h]: Ok

File: converter/parser.h (hunk 2 above)
Comment (evgenybritan) [reply converter/parser.h]: Yes, it is useless here

Comment (Java8Test) [reply converter/parser.h]: please update the code and update Pull Request.

File: converter/parser.h [hunk 3]
```diff
@@ -2,30 +2,14 @@
...
-  Gamut_Metadata_Packet,
-  InfoFrame_Packet
-};
+
 class Parser
 {
-
 public:
     Parser(QString);
-    void parse(QString);
```
Comment (Java8Test) [inline converter/parser.h]: Do we have at least one parse method?

**Comment by Java8Test** [comment]
approved as is. Ass global clean up expected

**Pull request merged**